### ISSUE TITLE
Unable to use this plugin alongside eslint plugin

### ISSUE THREAD

**Issue by apellerano-pw** [lifeart/vsc-ember-syntax#49]
If you use the ESLint VSCode plugin to highlight errors, this plugin will downgrade all your errors to warnings.

I think it's because of this unconditionally applied change to the user config on every boot.

https://github.com/lifeart/vsc-ember-syntax/blob/3921656e616580b9036b60a06abc1aa4fb436d64/src/index.ts#L28

Does this plugin require this config to work? Could it be made conditional and controllable by a setting? Could it be eliminated entirely?

**Comment by lifeart** [comment]
As possible fix I think we could avoid overriding and replace it with extending.
Smf like: 

```ts
const validate = eslintConfig.get('validate') ?? [];
const glimmerScopes = ['glimmer-ts', 'glimmer-js'];
eslintConfig.update('validate', Array.from(new Set([...validate, ...glimmerScopes])), vscode.ConfigurationTarget.Global);
```

**Comment by evoactivity** [comment]
I think that suggestion should be fine @lifeart. We might even consider removing it entirely to be honest.

I think the reason I added that at all was because after I upgraded my project and workspace to use gts etc and I was working on improving support I noticed everything that was a warning before was now an error (maybe correctly?).

**Comment by lifeart** [comment]
I'm fine with removal, since we already have it documented in readme (users could tweak eslint if needed)

**Comment by lifeart** [comment]
@apellerano-pw changes published [v1.0.17](https://github.com/lifeart/vsc-ember-syntax/releases/tag/v1.0.17)

**Comment by apellerano-pw** [comment]
Thank you, just tested the 1.0.17 version and it shows eslint errors as intended!
